doc/migration.rst

```diff
@@ -1,6 +1,190 @@
 :github_url: https://github.com/ros-controls/ros2_control/blob/{REPOS_FILE_BRANCH}/doc/migration.rst
 
-Migration Guides: Jazzy to Kilted
+Iron to Jazzy
```

Comment:
```suggestion
Humble to Jazzy
```

The release notes mentions Humble to Jazzy, so for consistency reasons I suggested this change